packages/amplify-category-storage/src/provider-utils/awscloudformation/service-walkthroughs/dynamoDb-walkthrough.ts

```diff
@@ -641,13 +650,13 @@ async function addTrigger(context: $TSContext, resourceName: string, triggerList
 async function getLambdaFunctions(context: $TSContext) {
   const { allResources } = await context.amplify.getResourceStatus();
   const lambdaResources = allResources
-    .filter((resource: any) => resource.service === FunctionServiceNameLambdaFunction)
+    .filter((resource: any) => resource.service === AmplifySupportedService.LAMBDA)
     .map((resource: any) => resource.resourceName);
 
   return lambdaResources;
 }
 
-function migrateCategory(context: $TSContext, projectPath: any, resourceName: any) {
+export function migrate(context: $TSContext, projectPath: any, resourceName: any) {
```

Comment:
why this name changed?

Reply:
`migrate` is what it was previously exported as in module.exports